src/script/lua_api/l_object.cpp

```diff
@@ -132,10 +132,22 @@ int ObjectRef::gc_object(lua_State *L) {
 int ObjectRef::l_remove(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
+	GET_ENV_PTR;
+
 	ObjectRef *ref = checkobject(L, 1);
 	ServerActiveObject *co = getobject(ref);
-	if(co == NULL) return 0;
-	if(co->getType() == ACTIVEOBJECT_TYPE_PLAYER) return 0;
+	if(co == NULL)
```

Comment:
Space, the new frontier. These are the voyages of the starship Enterprise. Its five-year missio...

Err... sorry. Ignore me, I was confused with something else.